src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -17,7 +17,11 @@ var _trans_icon = '';
 var _trans_link = '';
 var _trans_password = '';
 var _trans_drag = '';
-
+let tableEditorBeforeSaveEvent = new CustomEvent('tableEditorBeforeSaveEvent', {
```

Comment:
Line 20 should be const if `tableEditorBeforeSaveEvent` is not reassigned.

It's not immediately relevant to this PR, but should those variable declarations in lines 17-18 (et. al. the whole file has `var` declarations) also use the newer variable declaration keywords (const or let depending on reassignment; it seems these are also const, I'm not even sure that these are ever read (lines 17-19 that is).)?

Reply:
To not break any code, I will not do refactoring in this release. The whole backend JS code is very legacy and needs some love. But i will use const here for the added lines.

Reply:
Understood.